lib/index.js

```diff
@@ -114,9 +122,10 @@ exports.plugin = {
           if (currentCodeConfig && currentCodeConfig.redirect) {
             // if redirect is function invoke it with the request object
             if (isFunction(currentCodeConfig.redirect)) {
-              return reply.redirect(currentCodeConfig.redirect(request));
-            } 
-            else { 
+              const url = currentCodeConfig.redirect(request)
+              return reply.redirect(url);
```

Comment:
yes - it is easier to read

Reply:
I was debugging why redirect wasn't working in the test see below ⬇️ https://github.com/dwyl/hapi-error/pull/74/files/405fa01cfcffbaf8f37812f3b75a4f1947010a04#r372028342
and just decided to leave in this temporary variable. V8 will garbage collect it pretty quickly or might even in-line it in the JIT so I'm not concerned with perf. 👍